src/krate.rs

```diff
@@ -39,6 +39,8 @@ use util::{RequestUtils, CargoResult, internal, ChainError, human};
 use version::{EncodableVersion, NewVersion};
 use {Model, User, Keyword, Version, Category, Badge, Replica};
 
+const DOCUMENTATION_BLACKLIST: [&'static str; 1] = ["rust-ci.org"];
```

Comment:
What do you think about adding a doc comment for this constant explaining why a host might be in this list?